src/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -167,6 +167,7 @@ ascanbeta.sourcecodedisclosure.svnbased.extrainfo = The source code for [{0}] wa
 ascanbeta.sourcecodedisclosurewebinf.name = Source Code Disclosure - /WEB-INF folder
 ascanbeta.sourcecodedisclosurewebinf.desc = Java source code was disclosed by the web server in Java class files in the WEB-INF folder. The class files can be dis-assembled to produce source code which very closely matches the original source code.  
 ascanbeta.sourcecodedisclosurewebinf.soln = The web server should be configured to not serve the /WEB-INF folder or its contents to web browsers, since it contains sensitive information such as compiled Java source code and properties files which may contain credentials. Java classes deployed with the application should be obfuscated, as an additional layer of defence in a "defence-in-depth" approach.
+ascanbeta.sourcecodedisclosurewebinf.skipJava9 = not supported on Java 9+
```

Comment:
Should this have a leading capital? Do the other skip reasons?

Reply:
No, that's inserted into `Skipped, {0}.`.